_specs/ecip-1062.md

```diff
@@ -0,0 +1,59 @@
+---
+lang: en
+ecip: 1062
+title: Submit Block RPC
+author: Zachary Belford, zachary.b@etclabs.org
+discussions-to: Community Discord #ecips
+status: Draft
+type: Standards Track
+category: Interface
+created: 2019-06-26
+---
+
+### Abstract
+
+Add an RPC method to submit a new block, instead of having to use submitWork.
+
+### Motivation
+
+I would like to be able to submitWork, but have the address used be one other than the etherbase of the node the work is being submitted by.
+
+### Specification
+
+```json
+{
+  "openrpc": "1.1.8",
+  "info": {
+    "version": "1.0.0",
+    "title": "Example submitBlock"
+  },
+  "methods": [
+    {
+      "name": "eth_submitBlock",
+      "summary": "Submit a new block proposal",
+      "description": "Given a new block, it will validate the block, and broadcast it to connected peers.",
+      "params": [
+        { "$ref": "https://raw.githubusercontent.com/etclabscore/ethereum-json-rpc-specification/master/openrpc.json#/components/contentDescriptors/Block" }
+      ],
+      "result": {
+        "name": "isBlockAccepted",
+        "schema": {
+          "type": "boolean"
+        }
+      }
+    }
+  ]
+}
+```
```

Comment:
Please link where relevant to Open-RPC when showing a JSON Schema for the method that uses Open-RPC. 

I understand Open-RPC is used in Multi-Geth, but the ECIP should take into consideration non-Open-RPC clients as well for clarity such as Parity and others.

Also, would be good to figure out if Parity can adopt at least for the case of the "classic" network.